VirtoCommerce.Domain/Customer/Model/Organization.cs

```diff
@@ -1,11 +1,50 @@
-﻿namespace VirtoCommerce.Domain.Customer.Model
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using VirtoCommerce.Domain.Commerce.Model;
+using VirtoCommerce.Platform.Core.DynamicProperties;
+
+namespace VirtoCommerce.Domain.Customer.Model
 {
-    public class Organization : Member
+    public class Organization : Member, ICloneable
     {
         public string Description { get; set; }
         public string BusinessCategory { get; set; }
         public string OwnerId { get; set; }
         public string ParentId { get; set; }
 
+        public virtual object Clone()
+        {
+            var clone = (Organization) MemberwiseClone();
+
+            clone.Addresses = Addresses
+                ?.Select(x => (Address) x.Clone())
+                .ToList();
+
+            if (Phones != null)
+            {
+                clone.Phones = new List<string>(Phones);
```

Comment:
Better use ToList() here.

Reply:
Coding preference? The implementation of ToList does the same thing.

Reply:
Let's leave it then